src/components/task-plan/homework/add-exercises.cjsx

```diff
@@ -98,6 +98,8 @@ AddExercises = React.createClass
 
     sharedProps =
         exercises: exercises.all
+        watchStore: TaskPlanStore
+        watchEvent: 'change-exercise-'
         onExerciseToggle: @onExerciseToggle
```

Comment:
Will do, I'll remove from `sharedProps`.  I already tested without the select_count stuff there, and it seemed to be fine.